### DISCUSSION TITLE
Pengurusan Muaalaf

### DISCUSSION THREAD

**Discussion by rohaizanr** [Dev4w4n/e-masjid.my#208] [Ideas]
Open for ideas ..

**Comment by imie** [comment]
muallaf had limit dia 5 tahun sahaja. lepas 5 tahun da tak di kira sebagai muallaf. 

1. adakah perlukan pengesahan dari pejabat agama? 
2. bagaimana bantuan akan disalurkan?
3. tarikh muallaf dikira dari hari dia memeluk islam. bagaimana mekanisma untuk mengesahkan dia masih lagi berstatus muallaf?
4. ada keistimewaan untuk muallaf? contoh, kelas agama percuma, bimbingan solat, etc.

**Reply by lurhays** [reply]
Muallaf ada buddy system ke? Or assignment keluarga angkat or mentor. Aku tak berapa tahu sebab kecik sangat dulu, vaguely recall my late dad dulu ada guru, active perkim and join tabligh. But, tak sure macam mana role masjid dalam day to day life muallaf.
